ts/a11y/explorer/KeyExplorer.ts

```diff
@@ -207,6 +207,10 @@ export class SpeechExplorer extends AbstractKeyExplorer<string> {
   public Update(force: boolean = false) {
     super.Update(force);
     this.region.Update(this.walker.speech());
+    let options = this.speechGenerator.getOptions();
```

Comment:
Should this be just `this.getOptions()` like line 109 above?  Or does this not need the extra walker update from that function?

Reply:
No that part is necessary in case the speech options have changed by key pressing while walking the expression.  Merging in the `document` options would destroy the effect. I've added a comment to that extent.